tests/datatypes/test_bytes.py

```diff
@@ -20,6 +20,20 @@ def test_getattr(self):
                 print(err)
             """)
 
+    def test_capitalize(self):
+        self.assertCodeExecution("""
+            print(b'hello, world'.capitalize())
+            print(b'helloWORLD'.capitalize())
+            print(b'HELLO WORLD'.capitalize())
+            print(b'2015638687'.capitalize())
```

Comment:
I was watching over @katieamazing's shoulder while she was doing this and we already tried that, but we bumped into https://github.com/pybee/voc/issues/530 .

So perhaps it can be added in a separated test method and marked with `@expectedfailure` for now until issue #530 is fixed.